datasets/development_dataset_setup.py

```diff
@@ -0,0 +1,79 @@
+r"""MLCommons dataset setup script for development workloads.
```

Comment:
would it be possible to move this into `datasets/dataset_setup.py`? I don't think it's a big deal if we download mnist and cifar along with the other datasets if we use `--all` because theyre so tiny, but you could also exclude them from that if you want.

Reply:
That makes sense -- I will merge them!